packages/design-system-docs/src/scripts/components/Nav.jsx

```diff
@@ -30,11 +31,29 @@ function isParentOfSelectedChild(items, selectedId) {
   return false;
 }
 
+function updateItemsWithRootPath(items) {
```

Comment:
The reason why we didnt have to do this in the old scripts is because the `rootPath` used to be accounted for earlier on, it was taken into account in earlier on, [like here](https://github.com/CMSgov/design-system/pull/743/files#diff-2b20f8962ae43cec333c2999d49f8e02L49). This PR arguably makes the doc site react components more complex, because you need to handle `rootPath`, but IMO it also makes the doc generation scripts simpler as well. Instead of `rootPath` affecting the location of build files and some other internal data like `referenceURI`, we only use `rootPath` when generating markup, whether that's in the doc site react components, or in `generateHtmlExample.js` for example

Reply:
Also technically the doc site react components had to handle the `rootPath` with images as well anyway like [here](https://github.com/CMSgov/design-system/pull/743/files#diff-ffdd7c2f4e66cdd30639d9e5d788ff20L52), so 🤷